appinventor/components-ios/src/extensions/posenet.swift

```diff
@@ -0,0 +1,255 @@
+        }
+      }
+    }
+  }
+  
+  @objc open var MinPartConfidence: Double {
+    get {
+      return _minPartConfidence
+    }
+    set {
+      _minPartConfidence = newValue
+      if _initialized {
+        do {
+          try assertWebView("MinPartConfidence")
+        } catch {
+          print(_ERROR_WEBVIEWER_NOT_SET, "MinPartConfidence")
+        }
+      }
+    }
+  }
+  
+  @objc open var KeyPoints: [String] {
+    get {
+      var keyPoints: [[String]] = []
+      for point in _keyPoints.values {
+        if point.count == 2 {
+          keyPoints.append(point)
+        }
+      }
+      // TODO: build yaillist?
+      // HELP: what is YailList.makelist returning?
+      // returning flat array?
+      return keyPoints.flatMap { $0 }
+    }
+  }
+  
+  @objc open var Skeleton: [String] {
+    let lWrist = _keyPoints["leftWrist"]
+    let lElbow = _keyPoints["leftElbow"]
+    let lShoulder = _keyPoints["leftShoulder"]
+    let rShoulder = _keyPoints["rightShoulder"]
+    let rElbow = _keyPoints["rightElbow"]
+    let rWrist = _keyPoints["rightWrist"]
+    let lHip = _keyPoints["leftHip"]
+    let lKnee = _keyPoints["leftKnee"]
+    let lAnkle = _keyPoints["leftAnkle"]
+    let rHip = _keyPoints["rightHip"]
+    let rKnee = _keyPoints["rightKnee"]
+    let rAnkle = _keyPoints["rightAnkle"]
+    
+    // TODO: generate correct return value
+    return []
+  }
+  
+  // TODO: add all body part properties and find YailList representation
+  
+  
+  
+  
+  @objc open var BackgroundImage: String {
+      return _backgroundImage
+  }
+  
+  @objc open var Enabled: Bool {
+    get {
+      return _enabled
+    }
+    set {
+      _enabled = newValue
+      if _initialized {
+        do {
+          try assertWebView("Enabled")
+        } catch {
+          print(_ERROR_WEBVIEWER_NOT_SET, "Enabled")
+        }
+      }
+    }
+  }
+  
+  @objc open var UseCamera: String {
+    get {
+      return _cameraMode
+    }
+    set {
+      if newValue == _BACK_CAMERA || newValue == _FRONT_CAMERA {
+        _cameraMode = newValue
+        if _initialized {
+          do {
+            let frontFacing = (newValue == _FRONT_CAMERA)
+            try assertWebView("UseCamera", frontFacing)
+          } catch {
+            print(_ERROR_WEBVIEWER_NOT_SET, "UseCamera")
+          }
+        }
+      } else {
+          // TODO: figure out what form is
+      }
+    }
+  }
+  
+  
+  // MARK: PoseNet Methods
+  
+  
+  open func userContentController(
+    _ userContentController: WKUserContentController,
+    didReceive message: WKScriptMessage
+  ) {
+    if message.name == "ready" {
+      if _enabled {
+        MinPartConfidence = _minPartConfidence
+        MinPoseConfidence = _minPoseConfidence
+        UseCamera = _cameraMode
+      }
+    }
+    if message.name == "reportImage" {
+      // TODO: Finish out the javascript interface here
+    }
+  }
+  
+  @objc open func initialize() {
```

Comment:
The `Initialize` function is special and invoked by the runtime just before the Screen's `Initialize` method. However, the capitalization of the method name is important (the first I must be capital, not lowercase as you have here).